src/button/buttonview.js

```diff
@@ -18,8 +21,87 @@ export default class ButtonView extends View {
 	/**
 	 * @inheritDoc
 	 */
-	constructor() {
-		super();
+	constructor( locale ) {
```

Comment:
The second parameter with options (properties) would be nice, so I can create a button with label, instead of creating anonymous button and adding label later. 
```
const buttonView = new ButtonView( locale, {
    laber: 'foo',
    withText: true
} );
```